recipe/meta.yaml

```diff
@@ -23,8 +22,6 @@ build:
   number: 1
```

Comment:
should bump build num

Reply:
Thanks for the thorough review.

Reply:
Let's do that after we find that the tests pass or fail